app/models/image_data.rb

```diff
@@ -7,4 +7,11 @@ class ImageData < ApplicationRecord
 
   validates :file, presence: true
   validates_with ImageValidator, size: [960, 640]
+
+  def auth_bypass_ids
+    images
+      .filter { |image| image.respond_to?(:edition) }
+      .filter_map { |image| image.edition.auth_bypass_id if image.edition.pre_publication? }
```

Comment:
Do we need line 13 considering image belongs to edition?
Maybe worth testing something like `images.joins(:edition).where("edition.states in (?)", Edition::PRE_PUBLICATION_STATES).uniq`